includes/restapi/class-notification-controller.php

```diff
@@ -81,4 +91,180 @@ public function get_notifications( $request ) {
 
 		return rest_ensure_response( json_decode( $demo_data ) );
 	}
+
+	/**
+	 * Retrieves the notification's schema, conforming to JSON Schema.
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'notification',
+			'type'       => 'object',
+			'properties' => array(
+				'id'           => array(
+					'description' => __( 'Unique identifier for the notification.' ),
+					'type'        => 'integer',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'channel_id'   => array(
+					'description' => __( 'Unique identifier for the notification channel.' ),
+					'type'        => 'integer',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'created_at'   => array(
+					'description' => __( "The datetime the notification was broadcast, in the site's timezone." ),
+					'type'        => 'string',
+					'format'      => 'date-time',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'dismissed_at' => array(
+					'description' => __( "The datetime the notification was dismissed, in the site's timezone." ),
+					'type'        => array( 'string', 'null' ),
+					'format'      => 'date-time',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'displayed_at' => array(
+					'description' => __( "The datetime the notification was displayed, in the site's timezone." ),
+					'type'        => array( 'string', 'null' ),
+					'format'      => 'date-time',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'expires_at'   => array(
+					'description' => __( "The datetime the notification expires, in the site's timezone." ),
+					'type'        => 'string',
+					'format'      => 'date-time',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'context'      => array(
+					'description' => __( 'The view context the notification.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'default'     => 'hub',
+					'enum'        => array(
+						'adminbar',
+						'dashboard',
+						'hub',
+					),
+				),
+				'severity'     => array(
+					'description' => __( 'The severity of the notification.' ),
+					'type'        => array( 'string', 'null' ),
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'enum'        => array(
+						'alert',
+						'warning',
+						'success',
+					),
+				),
+				'status'       => array(
+					'description' => __( 'The status of the notification.' ),
+					'type'        => 'string',
+					'default'     => 'undisplayed',
+					'enum'        => array(
+						'undisplayed',
+						'displayed',
+						'dismissed',
+					),
+				),
```

Comment:
@erikyo how do these look to you?

Reply:
to me they are fine but it seems to me that it is missing `info` that is currently used with [WordPress](https://developer.wordpress.org/reference/hooks/admin_notices/)